src/CashuWallet.ts

```diff
@@ -218,32 +243,32 @@ class CashuWallet {
 		return { amount1, amount2 };
 	}
 
-	private async createRandomBlindedMessages(
+	private createRandomBlindedMessages(
 		amount: number
-	): Promise<BlindedMessageData & { amounts: Array<number> }> {
+	): BlindedMessageData & { amounts: Array<number> } {
 		const blindedMessages: Array<SerializedBlindedMessage> = [];
 		const secrets: Array<Uint8Array> = [];
 		const rs: Array<bigint> = [];
 		const amounts = splitAmount(amount);
 		for (let i = 0; i < amounts.length; i++) {
 			const secret = randomBytes(32);
 			secrets.push(secret);
-			const { B_, r } = await dhke.blindMessage(secret);
+			const { B_, r } = dhke.blindMessage(secret);
 			rs.push(r);
 			const blindedMessage = new BlindedMessage(amounts[i], B_);
 			blindedMessages.push(blindedMessage.getSerializedBlindedMessage());
 		}
 		return { blindedMessages, secrets, rs, amounts };
 	}
-	private async createBlankOutputs(feeReserve: number): Promise<BlindedMessageData> {
+	private createBlankOutputs(feeReserve: number): BlindedMessageData {
 		const blindedMessages: Array<SerializedBlindedMessage> = [];
 		const secrets: Array<Uint8Array> = [];
 		const rs: Array<bigint> = [];
 		const count = Math.ceil(Math.log2(feeReserve));
 		for (let i = 0; i < count; i++) {
 			const secret = randomBytes(32);
 			secrets.push(secret);
-			const { B_, r } = await dhke.blindMessage(secret);
+			const { B_, r } = dhke.blindMessage(secret);
```

Comment:
did this change because we use the new curve lib? or how come this is no longer async

Reply:
Yes